scripts/beck/st/error_injection.py

```diff
@@ -147,14 +159,14 @@ def recordOffset(record, i, size):
 
 def ReadTypeSize(byte):					#Ex: 0001(ver) 0009(type) 00003dd8(size) 
 	ver=byte[:4]						#.ot_version = src->hd_bits >> 48,
```

Comment:
s/byte[:4]/byte[0:4]/


No need to omit the starting index.

Reply:
I can not add comments to existing code. But I have comments too:

1. 
typeDict:
can we put one element on a separate line?
and other two dicts.

2.def recordOffset(record, i, size):
This function is ugly.

Can we do this:
```
def recordOffset(record, i, size):
        recordDict[record].append(i)
        sizeDict[record].append(size)
```

Maybe we need to assert the key exists in these two dicts.

Reply:
Let me test that 👍